project/Build.scala

```diff
@@ -38,7 +38,20 @@ import org.scalajs.core.tools.linker.backend.OutputMode
 
 import sbtassembly.AssemblyPlugin.autoImport._
 
-object Build extends sbt.Build {
+/*
+ * In sbt 0.13 the Build trait would expose all vals to the shell, where you can use them in "set a := b"
+ * like expressions. This re-exposes them.
```

Comment:
Good question. I think it's mostly habit in this case too, but it is a codified habit. After all, most coding style rules are habits. I could argue that it is more consistent with the Scaladoc comments, which also start on the first line, but I do not think that is why I started doing it that way in the first place.